tests/e2e/scenarios/iperf3-attached/inventory/host_vars/iperf3-client.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Kubeconfig path for west site
+client_kubeconfig: "{{ ansible_env.HOME }}/.kube/config"
```

Comment:
It might be better to standardize variable names, like in hello world.
I suggest using `kubeconfig_1` and `kubeconfig_2`.

Reply:
Fixed under: b498ed86